app/policies/draft_policy.rb

```diff
@@ -0,0 +1,35 @@
+# :nodoc:
+class DraftPolicy < ApplicationPolicy
+  def show?
+    user.user.provider_id == target.provider_id
```

Comment:
Why do these policies use `user.user.provider_id`?

Reply:
We add additional user context to pass in the token (for the ACL checks), in `def pundit_user` in `ApplicationController`. That adds another level in the object, so we need to use `user.user` to access the user and `user.token` to access the token.